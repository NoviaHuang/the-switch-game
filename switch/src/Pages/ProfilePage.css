* {
  font-family: 'Courier New', Courier, monospace;
}

.profile {
  background-color: #282c34;
  overflow: auto;
  height: 100vh;
}

.profile-img {
  height: 20%;
  width: auto;
  position: relative;
  top: 8em;
  left: 39%;
}

.profile-back-button {
  position: relative;
  left: 2%;
  top: 1em;
  bottom: 8em;
  background-color: #e7e7e7;
  color: black;
  border-radius: 8px;
  border-color: #e7e7e7;
  font-size: 100%;
  padding: 0.2% 1%;
  -webkit-transition-duration: 0.4s;
  transition-duration: 0.4s;
}

.profile-back-button:hover {
  background-color: rgb(93, 106, 134);
  color: white;
  border-color:rgb(93, 106, 134);
}

.username {
  color: rgb(235, 154, 108);
  position: relative;
  text-align: center;
  top: 15%;
}

.score {
  color: white;
  position: relative;
  text-align: center;
  top: 20%;
  font-size: 150%;
}

.passwordsetting{
  color: white;
  top: 25%;
  position: relative;
  text-align: center;

}
.oldpassword{
  margin: 1%;
}

.newpassword{
  margin: 1%;
}
.confirmpassword{
  margin: 1%;
}


.submitbutton{
  margin:1%;
  color: black;
  border-radius: 8px;
  border-color: #e7e7e7;
  font-size: 100%;
  -webkit-transition-duration: 0.4s;
  transition-duration: 0.4s;
}

