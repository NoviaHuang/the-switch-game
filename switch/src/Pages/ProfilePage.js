import React, { Component } from 'react';

class profile extends Component {
  render() {
    return (
        <div className="App">
        <div style={{width: '100%', margin: 'auto'}}>
    <Grid className="landing-grid">
      <Cell col={12}>
      <img
          src="https://www.shareicon.net/download/2015/09/18/103157_man_512x512.png"
          alt="profile-img"
          className="profile-img"
          />

        <div>
          <h1>username</h1>
        <hr/>
        </div>
      <p>Win: | Lost: | Presentage: |</p>
      {/* game records */}
      <div className="social-links">
    
      <div>
          <p>name</p>
          <input placeholder="username"></input>
          {/* username should placehold to the current name */}
      </div>
      <div>
          <p>Email</p>
          <input placeholder="email"></input>
          {/*email should placehold to the current name */}
      </div>
      <div>
          <p>Gender</p>
          <input required type="radio" value="male"></input>Male
          <input required type="radio" value="female"/>Female
      </div>
      <div>
          <p>Password</p>
          <input placeholder="8-10 chatacters"></input>
      </div>
      <div>
          <p>Confirm Password</p>
          <input></input>
      </div>
      <button type="submit">Save</button>
    </div>
        
      </Cell>
    </Grid>

        </div>
        </div>
    )
}
}

export default profile;