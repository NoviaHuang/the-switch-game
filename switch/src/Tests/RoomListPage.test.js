import 'react-testing-library/cleanup-after-each';
import 'jest-dom/extend-expect';
import React from 'react';
import ProfilePage from '../Pages/RoomListPage';
import {render} from 'react-testing-library'
const puppeteer = require('puppeteer');