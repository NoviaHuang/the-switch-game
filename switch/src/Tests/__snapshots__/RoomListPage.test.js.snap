// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RoomList Page screenshot test RoomList page should render correctly 1`] = `
<Route>
  [Function]
</Route>
`;
